src/test/java/com/mavenit/argos/SmokeTestSuite.java

```diff
@@ -0,0 +1,286 @@
+        System.out.println("'verifyProductRating' Test Passed - you can see all the below products has rating greater than or equal toe '5'");
+        List<WebElement> ratingWebElements = driver.findElements(By.cssSelector("div[data-test=\"component-ratings\"]"));
+        for (WebElement indProductRating : ratingWebElements) {
+            String currentRating = indProductRating.getAttribute("data-star-rating");
+            double currentRating1 = Double.parseDouble(currentRating);
+
+            // now to verify that previously selected rating is same for all current products
+            assertTrue("Current Products have different Rating ", Double.parseDouble(currentRating) > Double.parseDouble(selectedRating));
+            System.out.printf("\nThe Rating of individual product is :" + "%.2f", currentRating1);
+        }
+    }
+
+    @Test
+    public void basketTest() throws InterruptedException {
+
+        doSearch("tripods, monopods and cases");
+        Thread.sleep(4000);
+        List<WebElement> productWebElements = driver.findElements(By.cssSelector("a[data-test='component-product-card-title']"));
+        if (productWebElements.size() == 0) {
+            Assert.fail("No Products found with: " + "tripods, monopods and cases");
+        }
+
+        // TODO: 2020-02-08 this will be converted in future
+        Random random = new Random();
+        int randomNumber = random.nextInt(productWebElements.size()-1 );
+
+        WebElement selectedElement = productWebElements.get(randomNumber);
+        String selectedProductName = selectedElement.getText();
+        selectedElement.click();
+        Thread.sleep(2000);
+        addToBasket();
+        Thread.sleep(5000);
+        goToBasket();
+        Thread.sleep(2000);
+        String actual = getProductNameInBasket();
+        assertThat(actual, is(equalToIgnoringCase(selectedProductName)));
+
+        System.out.println("\n\n'basketTest' Test Passed - Actual selected product & the product in the basket is same");
+        System.out.println("\nTotal number of products are: " + productWebElements.size());
+        System.out.println("Index number of random product is : " + randomNumber);
+        System.out.println("\nActual selected product is: " + selectedProductName );
+        System.out.println("Product in the Basket is:   " + actual);
+    }
+
+    @Test
+    public void basketMultipleQuantityTest() throws InterruptedException {
+
+        doSearch("tripods, monopods and cases");
+        Thread.sleep(4000);
+        List<WebElement> productWebElements = driver.findElements(By.cssSelector("a[data-test='component-product-card-title']"));
+        List<WebElement> price = driver.findElements(By.cssSelector("div[class^=\"ProductCardstyles__PriceText-l8f8q8-14\"]"));
+        if (productWebElements.size() == 0) {
+            Assert.fail("No Products found with: " + "tripods, monopods and cases");
+        }
+        // TODO: 2020-02-08 this will be converted in future
```

Comment:
Remove random class from here and make it separate class like helpers class, as I can see u using random more than 1 and in future you will be using more

Reply:
done, i made separate random class and whenever i need this class i will call it